app/javascript/components/molecules/InlineModal.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const InlineModal = ({ children }) => {
+  return (
```

Comment:
since this is just a rename no need to change it, but how do you feel about defaulting to the expression syntax when there's no additional logic in the render body?

```
const InlineModal = ({ children }) => (
  <div className='app-editable padding-left--2x padding-right--2x'>
    {children}
  </div>
)
```

I think sometime I might migrate us off of standard so we can specify our own linting rules. We can start by using the standard eslint config which has all of the rules in `standard` but that way we can add or ignore as we want